client/src/containers/dropdown/component.tsx

```diff
@@ -0,0 +1,65 @@
+import { Menu } from '@headlessui/react';
+import Avatar from 'components/avatar';
+import StringAvatar from 'containers/string-avatar';
+import { signOut, useSession } from 'next-auth/react';
+
+/* const user = {
+  name: 'emily selman',
+  imageUrl: '/images/emily-selman.jpeg',
+  imageUrl: null,
+  email: 'nancygreene@gmail.com',
+}; */
+
+const Dropdown = () => {
```

Comment:
```suggestion
const UserDropdown = () => {
```
Dropdown is a very generic name that could collapse with other UI elements. Consider changing it. Also, rename the name of the folder